Jumpscale/clients/blockchain/tfchain/TFChainCapacity.py

```diff
@@ -63,7 +63,7 @@ def _grid_broker_pub_key(self):
             self._grid_broker_pub_key_ = vk.to_curve25519_public_key()
         return self._grid_broker_pub_key_
 
-    def reserve_s3(self, email, threebot_id, size=1, source=None, refund=None):
+    def reserve_s3(self, email, threebot_id, location, size=1, source=None, refund=None):
```

Comment:
location is a bad name imho, would never have guessed that it would indicate the name of the farm. Also why not the organization IYO instead of the name?

Reply:
It is actually the farm ID, so the IYO organization. I need to update the documentation about it.
And I use a generic name as location cause it can be a farm name, or a node id.